icon/internal/_icon.scss

```diff
@@ -27,15 +27,16 @@
 
 @mixin styles() {
   $tokens: tokens.md-comp-icon-values();
+  @each $token, $value in $tokens {
+    $tokens: map.set($tokens, $token, var(--md-icon-#{$token}, #{$value}));
+  }
 
   :host {
-    font-size: 24px;
-    width: 24px;
-    height: 24px;
```

Comment:
Width and height are still required to properly size `<md-icon><svg>...` icons. It's also used to give font icons a default size so that they don't shift layout when they load in.

```
font-size: map.get($tokens, 'size');
width: map.get($tokens, 'size');
height: map.get($tokens, 'size');
```